test/project_types/script/commands/enable_test.rb

```diff
@@ -220,6 +217,40 @@ def test_calls_application_enable_with_configuration_file_and_properties_overrid
         end
       end
 
+      def test_calls_application_enable_with_invalid_configuration_file
+        File.open("enable_config_file.yml", "w+") { |file| file.write("key1 \"value1\"\nkey2: \"value2\"") }
+
+        Script::UI::ErrorHandler
+          .expects(:display)
+          .with(
+            failed_op: nil,
+            cause_of_error: ShopifyCli::Context.message('script.error.invalid_config', "enable_config_file.yml"),
+            help_suggestion: nil,
+          )
```

Comment:
If anyone can please let me know if there is a better way of `expects` this, as I couldn't figure it out.  What I wanted to do is something like 
```
Script::UI::ErrorHandler
          .expects(:pretty_print_and_raise).with(Errors::InvalidConfigYAMLError, " ...")
```
But it always tries to compare the instances of the `Errors::InvalidConfigYAMLError` objects.  I really want to use something like `instance_of(Errors::InvalidConfigYAMLError)` or something like that.  Anyone know how to do this better?

Reply:
You can do something like this:

```
Script::UI::ErrorHandler
          .expects(:pretty_print_and_raise)
          .with { |e, _message| e.is_a?(Errors::InvalidConfigYAMLError) }
```